lib/msf/ui/console/command_dispatcher/core.rb

```diff
@@ -56,7 +55,7 @@ class Core
     ["-k", "--kill"]            => [ true,  "Terminate the specified thread ID.", "<id>"             ],
     ["-K", "--kill-all"]        => [ false, "Terminate all non-critical threads."                    ],
     ["-i", "--info"]            => [ true,  "Lists detailed information about a thread.", "<id>"     ],
-    ["-l", "--list"] => [ false, "List all background threads."                           ],
+    ["-l", "--list"]            => [ false, "List all background threads."                           ],
     ["-v", "--verbose"]         => [ false, "Print more detailed info.  Use with -i and -l"          ])
```

Comment:
I can't comment on the line directly, but updating the `--search` options above:

```ruby
    ["-S", "--search"]               => [ true,  "Row search filter.", "<filter>"                                                ],
```

To include an example of the filter would be useful for users to discover this functionality :+1:

The thought being - if the feature isn't documented for users, it doesn't necessarily exist

Reply:
Good call!